src/Ledger/Utxo.lagda

```diff
@@ -205,9 +209,9 @@ module _ (let open Tx; open TxBody; open TxWitnesses) where opaque
 \end{NoConway}
 \begin{code}
   refScriptsSize : UTxO → Tx → ℕ
-  refScriptsSize utxo tx = ∑[ x ← mapValues scriptSize (setToHashMap (refScripts tx utxo)) ] x
+  refScriptsSize ux tx = ∑[ x ← mapValues scriptSize (setToHashMap (refScripts tx ux)) ] x
```

Comment:
We typically call `UTxO` variables `utxo` everywhere, so this change makes it inconsistent. It's also not a long variable name, so I think it's better to revert this line.

Reply:
I changed it to `ux` because saving those two characters makes the line fit inside the figure without splitting it up into multiple lines... but I'll change it back.